docs/pages/_document.js

```diff
@@ -1,3 +1,3 @@
-import MyDocument from '@material-ui/monorepo/docs/pages/_document';
+import MyDocument from 'docs/pages/_document';
```

Comment:
What do you think about using `'docs/'` as little as possible (only to avoid import crashes for the import from the main repo). I find it confusing that a file import itself (it seems so)

```suggestion
import MyDocument from '@mui/monorepo/docs/pages/_document';
```

Reply:
I replaced in all files that were already modified. The remaining will be updated latter.